### PR TITLE
[#658] Update participantDetail page && participant Summary page

### PR DESCRIPTION
**Pull request by jhflorey**
This PR address issue https://github.com/episphere/connect/issues/658

- Updated information of Participant Detail page
- Updated information of Participant Summary page

### CODE REVIEW

File: siteManagerDashboard/fieldToConceptIdMapping.js [hunk 1]
```diff
@@ -373,7 +373,7 @@ export default
     458508122 : "Refused all future activities",
     872012139 : "Revoked HIPAA only",
     854021266 : "Withdrew Consent",
-    241236037 :  "Destroy Data",
```
Comment (abhinavjonnada82) [inline siteManagerDashboard/fieldToConceptIdMapping.js:376]: Is this the new Destroy Data CID?
Did you update all the references?

File: siteManagerDashboard/participantSummaryRow.js [hunk 2]
```diff
@@ -42,10 +42,13 @@ export const verificationStatus = (participant) => {
 }
 
 export const baselineBloodSample = (participantModule) => {
+    const dataHasBeenDestroyed = participantModule[fieldMapping.dataHasBeenDestroyed]
     let refusedBloodOption = participantModule[fieldMapping.refusalOptions]?.[fieldMapping.refusedBlood];
```
Comment (abhinavjonnada82) [inline siteManagerDashboard/participantSummaryRow.js:46]: Optional: Could you add a comment to change from `let` to `const` as a reminder. 
This was my change & I'm not sure why I used `let`. I'll address them in my future PR.

**Comment by abhinavjonnada82** [comment]
@jhflorey Minor suggestions; otherwise, the code looks good, and thanks for integrating your changes into the existing code. Instead of changing the whole code.

**Review by abhinavjonnada82 — APPROVED** [review]

**Review by we-ai — APPROVED** [review]
Looks good.

**Pull request merged**